ACHNBrowserUI/ACHNBrowserUI/models/Category.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  Categories.swift
+//  Category.swift
```

Comment:
I think it should have been named Category right?. and `Categoriesa is when you have an array of categories `let categories: [Category]`

Reply:
Yep